api/app/GraphQL/Directives/SelfDirective.php

```diff
@@ -10,7 +10,7 @@
 
 final class SelfDirective extends BaseDirective implements FieldResolver
 {
-    // TODO implement the directive https://lighthouse-php.com/master/custom-directives/getting-started.html
+    // TODO: implement the directive https://lighthouse-php.com/6/custom-directives/getting-started.html#implementing-your-own-directives
```

Comment:
I suppose the comment is useful but I don't think it needs to be a TODO anymore 😆

Reply:
Updated in: https://github.com/GCTC-NTGC/gc-digital-talent/pull/9613/commits/100a5925d93cd1d9fe25e026ebf01a3be94cba1d